OptimizelySDK.Tests/OptimizelyConfigTests/OptimizelyConfigTest.cs

```diff
@@ -177,6 +176,33 @@ public void TestGetOptimizelyConfigSDKAndEnvironmentKeyDefault()
             Assert.AreEqual(optimizelyConfig.EnvironmentKey, "");
         }
 
+        [Test]
+        public void TestGetOptimizelyConfigWithOdpIntegration()
+        {
+            var datafileProjectConfig = DatafileProjectConfig.Create(TestData.TypedAudienceDatafile, new NoOpLogger(), new ErrorHandler.NoOpErrorHandler());
+
+            Assert.AreEqual("https://api.zaius.com", datafileProjectConfig.HostForOdp);
+            Assert.AreEqual("W4WzcEs-ABgXorzY7h1LCQ", datafileProjectConfig.PublicKeyForOdp);
+        }
+        
+        [Test]
+        public void TestGetOptimizelyConfigWithEmptyIntegrationCollection()
+        {
+            var datafileProjectConfig = DatafileProjectConfig.Create(TestData.EmptyDatafile, new NoOpLogger(), new ErrorHandler.NoOpErrorHandler());
+
+            Assert.IsNull(datafileProjectConfig.HostForOdp);
+            Assert.IsNull(datafileProjectConfig.PublicKeyForOdp);
+        }
+        
+        [Test]
+        public void TestGetOptimizelyConfigWithOtherIntegrationsInCollection()
+        {
+            var datafileProjectConfig = DatafileProjectConfig.Create(TestData.EmptyRolloutDatafile, new NoOpLogger(), new ErrorHandler.NoOpErrorHandler());
```

Comment:
Can we also have a test with arbitary fields in integration for validating forward compatibilty, like {"key": "any-key", "k1": 10, "k2": true, "k3": "any-value"}

Reply:
I did this in \OptimizelySDK.Tests\typed_audience_datafile.json  which is used in OptimizelySDK.Tests/OptimizelyConfigTests/OptimizelyConfigTest.cs

Should I move all these Integration tests in a single test file?  @jaeopt

Reply:
Yes, I saw that test in EmptyRolloutRule. It'll be more clear if we can put integrations testing in its won datafile.
Also it'll be good to have arbitrary fields in addition to existing "publicKey" and "host".